### PR TITLE
New package r-qs at 0.23.5

### PR DESCRIPTION
**Pull request by dorton21**

### CODE REVIEW

**Comment by dorton21** [comment]
depends on #21780

**Comment by dorton21** [comment]
depends on #21785

File: var/spack/repos/builtin/packages/r-qs/package.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    url      = "https://cloud.r-project.org/src/contrib/qs_0.23.5.tar.gz"
+    list_url = "https://cloud.r-project.org/src/contrib/Archive/qs"
+
+    maintainers = ['dorton21']
+
+    version('0.23.6', sha256='c6e958e9741ee981bf2388c91b8f181718ffb0f32283cd7ebcd2d054817280e4')
+
+    depends_on('r@3.5.0:', type=('build', 'run'))
+    depends_on('r-rcpp', type=('build', 'run'))
+    depends_on('r-rapiserializer', type=('build', 'run'))
```
Comment (glennpj) [inline var/spack/repos/builtin/packages/r-qs/package.py]:
```suggestion
    depends_on('r-rapiserialize', type=('build', 'run'))
```

**Review by glennpj — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/r-qs/package.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+from spack import *
+
+
+class RQs(RPackage):
+    """Quick Serialization of R Objects
+
+    Provides functions for quickly writing and reading any R object
+    to and from disk."""
+
+    homepage = "https://cran.r-project.org/web/packages/qs/index.html"
```
Comment (glennpj) [inline var/spack/repos/builtin/packages/r-qs/package.py]:
```suggestion
    homepage = "https://cloud.r-project.org/web/packages/qs/index.html"
```

**Review by glennpj — CHANGES_REQUESTED** [review]

**Review by glennpj — APPROVED** [review]

**Pull request merged**